modules/dop/initialize.go

```diff
@@ -592,3 +596,70 @@ func copyTestFileTask(ep *endpoints.Endpoints) {
 	}
 	ep.TestSetService().CopyTestSet(record)
 }
+
+// updateCmsNsConfigs update pipeline cms according to pipeline cron which enable is true
+// it will be deprecated in the later version
+func updateCmsNsConfigs(ep *endpoints.Endpoints) error {
+	enable := true
+	// get total
+	cron, err := bdl.Bdl.PageListPipelineCrons(apistructs.PipelineCronPagingRequest{
+		AllSources: true,
```

Comment:
No need all source. Maybe `dice` is enough.

Reply:
done